tests/v2/validation/rbac/etcdbackup/etcd_backup_test.go

```diff
@@ -0,0 +1,115 @@
+//go:build (validation || infra.any || cluster.any || stress) && !sanity && !extended
+
+package rbac
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/etcdsnapshot"
+	rbac "github.com/rancher/rancher/tests/framework/extensions/rbac"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ETCDRbacBackupTestSuite struct {
```

Comment:
this camelCase is hard to read at first glance. maybe all of rbac should be lowercase?

Reply:
maybe RBACSnapshotBackupTestSuite. saying just 'etcd' is sort of a norman / legacy way of referring to it.